src/main/java/io/split/android/client/events/SplitEventsManager.java

```diff
@@ -25,7 +30,7 @@
 
 public class SplitEventsManager implements ISplitEventsManager, Runnable {
 
-    private final static int QUEUE_CAPACITY = 10;
+    private final static int QUEUE_CAPACITY = 20;
```

Comment:
It might be a good idea to replace ArrayBlockingQueue for a LinkedBlockingQueue to avoid this constant. Something to consider in the future.

Reply:
Yup. Will. take this into account for technical debt.